packages/template-retail-react-app/package.json

```diff
@@ -26,7 +26,7 @@
         "base64-arraybuffer": "^0.2.0",
         "bundlesize2": "^0.0.31",
         "card-validator": "^8.1.1",
-        "commerce-sdk-isomorphic": "^1.9.0",
+        "commerce-sdk-isomorphic": "^1.12.0",
```

Comment:
Needs to use a newer version of commerce-sdk-isomorphic that supports custom parameter (for us to pass in our own `c_cache_breaker`).